src/core/server/http/http_config.ts

```diff
@@ -106,6 +108,17 @@ export const config = {
         if (!rawConfig.basePath && rawConfig.rewriteBasePath) {
           return 'cannot use [rewriteBasePath] when [basePath] is not specified';
         }
+
+        if (rawConfig.publicBaseUrl) {
+          const parsedUrl = url.parse(rawConfig.publicBaseUrl);
```

Comment:
We don't need to try/catch the `url.parse` call because we are using `schema.uri`, right?

Reply:
Yeah I'm relying on config-schema/Joi here implicitly.